### PR TITLE
update alerts; use slider; allow more detail with modal

### PR DESCRIPTION
**Pull request by loomis**

### CODE REVIEW

File: src/cljs/sixsq/slipstream/webui/main/views.cljs [hunk 1]
```diff
@@ -84,7 +84,8 @@
          [ui/MenuItem {:fitted true}
           [authn-views/authn-menu]]]]
 
-       [messages/message-modal]])))
+       [messages/alert-slider]
+       [messages/alert-modal]])))
```
Comment (0xbase12) [inline src/cljs/sixsq/slipstream/webui/main/views.cljs]: what is alert-modal for?

File: src/cljs/sixsq/slipstream/webui/messages/views.cljs [hunk 2]
```diff
@@ -40,66 +39,105 @@
...
+                      :size     "tiny"
+                      :icon     "close"
+                      :on-click (fn [e]
+                                  (when e
+                                    (.stopPropagation e true))
+                                  (dispatch [::message-events/remove index]))}]
+          [ui/ListHeader header]
+          [ui/ListDescription (time/ago timestamp locale)]]
+
+         [ui/Modal
```
Comment (0xbase12) [inline src/cljs/sixsq/slipstream/webui/messages/views.cljs]: click on modal cross icon to close doesn't work, escape key work

File: src/cljs/sixsq/slipstream/webui/main/views.cljs (hunk 1 above)
Comment (loomis) [reply src/cljs/sixsq/slipstream/webui/main/views.cljs]: The slider just provides the title and status.  The details can be found in the modal by clicking the slider notification or the entry in the list on the messages page.

Comment (loomis) [reply src/cljs/sixsq/slipstream/webui/main/views.cljs]: BTW, the details are important when, for example, creating an API key/secret pair.

File: src/cljs/sixsq/slipstream/webui/messages/views.cljs [hunk 3]
```diff
@@ -40,66 +39,105 @@
...
+    (fn []
+      (if-let [{:keys [type header]} @alert-message]
+        (let [icon-name (type->icon-name type)
+              open? (boolean (and @alert-message (= :slider @alert-display)))
+              transition (clj->js {:animation "slide left"
+                                   :duration  500})
+              top-right {:position "fixed", :top "0", :right "0", :zIndex 1000}]
+          [ui/TransitionablePortal {:transition transition, :open open?}
+           [ui/Message {:size       "mini"
+                        :success    true
```
[the remark this answers is not in the document]
Comment (loomis) [reply src/cljs/sixsq/slipstream/webui/messages/views.cljs]: That's a bug that needs to be corrected.

File: src/cljs/sixsq/slipstream/webui/messages/views.cljs (hunk 2 above)
Comment (loomis) [reply src/cljs/sixsq/slipstream/webui/messages/views.cljs]: Will check why this is happening.

**Comment by loomis** [comment]
You had something in mind like the notification widget in Freshdesk?

**Comment by 0xbase12** [comment]
Yes like freshdesk

**Comment by loomis** [comment]
@0xbase12 Try the latest changes.

**Pull request merged**